src/buildCommon.js

```diff
@@ -246,9 +246,9 @@ const makeOrd = function<NODETYPE: "spacing" | "mathord" | "textord">(
     const fontOrFamily = isFont ? options.font : options.fontFamily;
     if (value.charCodeAt(0) === 0xD835) {
         // surrogate pairs get special treatment
-        const [wideFontName, wideFontClass] = wideCharacterFont(value, mode);
```

Comment:
Babel's output for restructuring seems to be pretty short.  How much of a savings is this?

Reply:
@kevinbarabash The last time I checked, it included Iterator and Symbol polyfill, but now I check it seems to be short. I’ll check again and revert if necessary.

Reply:
@kevinbarabash It seems something got mixed up at the last time. The loose mode assumes right hand sided to be an array. I reverted relevant commit. Thank you!